examples/docs_beta_snippets/docs_beta_snippets/guides/components/shell-script-component/with-custom-udf.py

```diff
@@ -27,7 +27,7 @@ class ShellCommand(Component, ResolvedFrom[ShellCommandModel]):
     asset_specs: AssetSpecSequenceField
 
     @classmethod
-    def get_additional_scope(cls) -> Mapping[str, Any]:
+    def get_udfs(cls) -> Mapping[str, Any]:
         return {
             "daily_partitions": dg.DailyPartitionsDefinition(start_date="2024-01-01")
```

Comment:
`daily_partitions` feels inappropriate to label as a udf to me

Reply:
We can make everything a function by convention

Reply:
Does this include existing scopes, e.g. `node` in the dbt case?

`key: {{ get_node().name }}` is meaningfully worse than `key: {{ node.name }}` IMO, but maybe worth the simplification